wordpress-mobile/WordPress-iOS
# [Stats Refresh] remove unused code

Fixes #n/a

This just removes some code that is no longer used by Stats Refresh. There are no functional changes.

To test:
Basic sanity check - access Stats Insights & Period views. Verify all cards appear correctly.

Update release notes:
